web/src/components/tasks/business-formation/BusinessFormation.test.tsx

```diff
@@ -754,10 +760,13 @@ describe("<BusinessFormation />", () => {
     const legalStructureId = "limited-partnership";
     const profileData = generateFormationProfileData({ legalStructureId });
     const formationData = generateEmptyFormationData();
+    formationData.formationFormData.addressMunicipality = generateMunicipality({
```

Comment:
boulder - here is another example. I think every test you refactored needs to be revisited to confirm it ovrerriding the values is the right approach

Reply:
Reset to original and added appropriate fix.